packages/nimble-components/src/icon-base/tests/icon-metadata.ts

```diff
@@ -31,9 +31,15 @@ export const iconMetadata: {
     IconArrowExpanderUp: {
         tags: []
     },
+    IconArrowInCircle: {
+        tags: ['move in']
```

Comment:
```suggestion
        tags: ['move in', 'waka waka']
```

Reply:
Once you see pacman, you can't unsee pacman! I'll definitely steal this joke in my next Nimble update video but gonna leave it out of the repo :-P